### ISSUE TITLE
Detect if two servers are sending data for the same uuid

### ISSUE THREAD

**Issue by AuroraLS3** [plan-player-analytics/Plan#2049]
### I would like to be able to..

Detect and log a warning if two servers are sending data for the same server uuid

### Is your feature request related to a problem? Please describe.

One of the symptoms is that the players online graph is showing smaller than usual numbers due to highcharts optimizing the constant up/down of 2 server's data as the average of the two.

This can happen if a duplicated test setup can connect to the main database.

**Comment by rymiel** [comment]
Could this simply by detected as the frequency of the entries in database (according the timestamp) being higher than they should be or is that too unstable

**Comment by AuroraLS3** [comment]
Yeah that would work as long as the server has stored one entry before it checks the time since last entry
